packages/shared/src/error.ts

```diff
@@ -0,0 +1,63 @@
+const ERROR_CODES = {
+  INTERNAL_SERVER_ERROR: 500,
+  INTERNAL_CLIENT_ERROR: 500,
+  FAILED_TO_SIGN: 500,
+  FAILED_TO_UPLOAD: 500,
```

Comment:
I feel like I would prefer `<SOMETHING>_FAILED` over `FAILED_TO_<SOMETHING>`. 

I am not sure if my opinion here is bad practice though.